source/main.cpp

```diff
@@ -56,30 +55,31 @@ int main()
     app_logic *g_logic = new app_logic();
     g_logic->init();
 
-    Graphics::g_RenderCore.Set2DMode();
-    Graphics::g_RenderCore.SetClearColor(30, 30, 30, 255);
+    GFX::g_RenderCore->setClearColor(30.0f/255.0f, 30.0f/255.0f, 30.0f/255.0f, 1.0f);
 
 #ifdef INTERNAL_DEV
     const char* dev_ver = "DEV VERSION";
-    Graphics::UI::UIText* dev_version_text = new Graphics::UI::UIText({480 - (int)(intraFontMeasureText(Graphics::UI::g_DefaultFont, dev_ver) * 0.7f), 267}, dev_ver);
-    dev_version_text->setOptions({0.7f, 0xFFFFFFFF, INTRAFONT_ALIGN_LEFT});
+    
+    g::font_renderer = new GFX::UI::TextRenderer();
 #endif
 
     while (g_logic->is_running())
     {
-        Graphics::g_RenderCore.BeginCommands();
-        Graphics::g_RenderCore.Clear();
+        GFX::g_RenderCore->beginFrame();
+        GFX::g_RenderCore->clear();
 
         g_logic->run();
 
 #ifdef INTERNAL_DEV
-        dev_version_text->draw();
+        g::font_renderer->setStyle({255, 255, 255, 255, 0.7f, -1, INTRAFONT_ALIGN_RIGHT, 0, false});
```

Comment:
this style (align_right) would be persistent across every text, lets change it back to align_center after we draw

Reply:
You only set style before you draw.